pkg/config/config.go

```diff
@@ -88,3 +87,26 @@ func detectGoVersion() string {
 
```

Comment:
`runtime.Version()` would not be helpful here?

Reply:
The goal is to get the Go version of the project to compare it with `runtime.Version()`.